_includes/syllabus.html

```diff
@@ -78,16 +78,6 @@ <h2 id="schedule">Sample Schedule</h2>
   {% assign hours = current | divided_by: 60 %}
   {% assign minutes = current | modulo: 60 %}
   {% if site.instructor_training %}
-  <tr>
-    {% if multiday %}<td class="col-md-1"></td>{% endif %}
-    <td class="{% if multiday %}col-md-1{% else %}col-md-2{% endif %}">{% if hours < 10 %}0{% endif %}{{ hours }}:{% if minutes < 10 %}0{% endif +++ %}{{ minutes }}</td>
-    <td class="col-md-3"><a href="{{ site.instructor_post_survey }}{{ site.github.project_title }}">Post-training survey</a></td>
-    <td class="col-md-7">Please fill out our post-training survey after the course.</td>
-  </tr>
-  {% assign current = current | plus: 15 %}
-  {% assign hours = current | divided_by: 60 %}
-  {% assign minutes = current | modulo: 60 %}
-  {% endif %}
```

Comment:
this line closes the if that's started on line 80, that's the error in the build